site/layouts/shortcodes/include.html

```diff
@@ -1,6 +1,6 @@
 {{ $file := .Get 0 }}
 [{{$file}}]({{printf "/examples/%s" $file}})
-{{ $path := (printf "../../static/examples/%s" $file)}}
+{{ $path := (printf "/opt/build/repo/site/static/examples/%s" $file)}}
 ```{{ .Get 1 }}
-{{ (printf "%s%s" .Page.File.Dir $path) | readFile | replaceRE "^---[\\s\\S]+?---" "" | safeHTML }}
+{{ (printf "%s" $path) | readFile | replaceRE "^---[\\s\\S]+?---" "" | safeHTML }}
```

Comment:
```suggestion
{{ $path | readFile | replaceRE "^---[\\s\\S]+?---" "" | safeHTML }}
```
?

Reply:
This didn't work :(

https://app.netlify.com/sites/kubernetes-sigs-kueue/deploys/64dd2124fb60a8000848c12b